### PR TITLE
Arrange forc.toml fields alphabetically

### LINKED ISSUE #724: `Forc.toml` created by `forc init` isn't properly formatted
- Fields are not alphabetized
- This should ideally be fixed along #687

### PR DESCRIPTION
**Pull request by eureka-cpu**
closes #724 by arranging forc.toml fields alphabetically

### CODE REVIEW

File: forc/src/utils/defaults.rs [hunk 1]
```diff
@@ -5,10 +5,10 @@ pub(crate) fn default_manifest(project_name: &str) -> String {
 
     format!(
         r#"[project]
-name = "{}"
 author = "{}"
 entry = "main.sw"
 license = "Apache-2.0"
+name = "{}"
```
Comment (adlerjohn) [inline forc/src/utils/defaults.rs:11]: When you change the order of `{}`, you have to also change what value they're assigned!

File: forc/src/utils/defaults.rs [hunk 2]
```diff
@@ -27,11 +27,11 @@ pub(crate) fn default_tests_manifest(project_name: &str) -> String {
 
     format!(
         r#"[package]
-name = "{}"
-version = "0.1.0"
 authors = ["{}"]
 edition = "2021"
 license = "Apache-2.0"
+name = "{}"
```
Comment (adlerjohn) [inline forc/src/utils/defaults.rs:33]: Ditto

**Review by adlerjohn — CHANGES_REQUESTED** [review]

File: forc/src/utils/defaults.rs (hunk 1 above)
Comment (eureka-cpu) [reply forc/src/utils/defaults.rs:11]: Ah, I see, as in the name `{}` is not getting passed to the author's `{}`

**Review by adlerjohn — COMMENTED** [review]
Looks fine to me, but will defer to @sezna

**Review by adlerjohn — APPROVED** [review]

**Review by sezna — APPROVED** [review]

**Pull request merged**